app/views/contexts/summary_context.py

```diff
@@ -66,15 +115,62 @@ def get_title_for_location(self, location):
                 )
         return title
 
-    def section_summary(self, current_location):
-        block = self._schema.get_block(current_location.block_id)
+    def _custom_summary_elements(self, section_summary, current_location, section):
+        for summary_element in section_summary:
+            if summary_element["type"] == "List":
+                yield self._list_summary_element(
+                    summary_element, current_location, section
+                )
+
+    def _list_summary_element(
+        self, summary: Mapping, current_location, section: Mapping
+    ) -> Iterator[Mapping]:
+        list_collector_block = self._schema.get_list_collectors_for_section(
+            section, for_list=summary["for_list"]
+        )[0]
+
+        add_link = self._add_link(
+            summary, current_location, section, list_collector_block
+        )
+
+        rendered_list_context = self.list_context(
+            list_collector_block, current_location.block_id
+        )
```

Comment:
Doesn't this render the list items according to the `summary` element on the list collector block, rather than using the `item_title` element of the section summary?